app/models/project.rb

```diff
@@ -157,6 +157,8 @@ class Project < ApplicationRecord
   scope :visible, ->(user = User.current) { where(id: Project.visible_by(user)) }
   scope :newest, -> { order(created_at: :desc) }
   scope :active, -> { where(active: true) }
+  scope :with_member, ->(user = User.current) { where(id: user.memberships.pluck(:project_id)) }
+  scope :without_member, ->(user = User.current) { where.not(id: user.memberships.pluck(:project_id)) }
```

Comment:
Using `pluck` in this context leads to an unnecessary extra request as the `pluck` has to be resolved first with the results of that then being passed into the next query. Using `select` avoids that.